MdocDataTransfer.podspec

```diff
@@ -21,4 +21,15 @@ Pod::Spec.new do |s|
 
   s.source_files = "ios/**/*.{h,m,mm,swift}"
 
+  install_modules_dependencies(s)
+
+  if defined?(:spm_dependency)
+    spm_dependency(s,  
+      url: 'https://github.com/berendsliedrecht/eudi-lib-ios-iso18013-data-transfer.git', 
```

Comment:
Might be good to add a comment with reason and link to the code: https://github.com/berendsliedrecht/eudi-lib-ios-iso18013-data-transfer/tree/send-device-response-manually

Also -- maybe good to clean up the formatting in that branch a bit so it's easy to see what changed compared to the EUDI wallet implementation.